### DISCUSSION TITLE
移动端自动化操作：maestro

### DISCUSSION THREAD

**Discussion by adoyle-h** [just-talks/tech-talks#17] [3.分享创造]
[maestro](https://github.com/mobile-dev-inc/maestro) 在移动端执行自动化操作。

使用声明式语法，降低了学习使用门槛。

![gif](https://user-images.githubusercontent.com/847683/187275009-ddbdf963-ce1d-4e07-ac08-b10f145e8894.gif)

**Comment by gilbertwong96** [comment]
404 not found

**Reply by adoyle-h** [reply]
已修正
